Sponsors.md

```diff
@@ -80,6 +79,18 @@ The list below describes the awareness benefits that our Logo sponsor receives i
 * “Thank you” mention in facebook & Twitter page post,
 * “Thank-you” message by organizers during Meetup (not by company/individual).
 
+## Equipment Sponsor
+As we are not close to the local community of Crete we try to live stream our monthly meetups and supply VoD through Youtube. FOr this reason we use equipment which mosto of the times is not suitable for our need.
```

Comment:
** For this reason we use equipment which most of **

Reply:
Fixed by https://github.com/devstaff-crete/DevStaff-Heraklion/pull/413/commits/2b308bbb5a661f83a7deec7f2a009fb3a90f38fa